controller/class.tapestry-controller.php

```diff
@@ -43,6 +43,10 @@ class TapestryController
         'INVALID_CHILD_NODE' => [
             'MESSAGE'   => 'Node is not a child of the tapestry',
             'STATUS'    => ['status' => 400]
+        ],
+        'INVALID_TAPESTRY' => [
```

Comment:
The key here is not descriptive of the error. Maybe `INVALID_NEW_TAPESTRY` would be better.